CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added Go config.
```

Comment:
If I remember correctly, the consensus was that we don't add modifications to the changelog if the change is not relevant/visible to our users, so I guess we'll add a single entry for the whole go binary, when it becomes ready for production. You can remove this and just add the `no-changelog` tag 😉

Reply:
@gligneul now you got me confused. Should or shouldn't we add single container updates to the changelog? 🤔

Reply:
If it is in the unreleased section, I see no harm in adding them now.